Makefile

```
@@ -1,17 +1,16 @@
-all: install tests
+mutation: install
+	mutmut run --paths-to-mutate nerd
 
 .PHONY: \
-    all \
-    build \
-    build_demo \
     clean \
     install \
     lint \
-    run_demo \
+    mutation \
     tests \
 
-build:
-	docker build --tag=islasgeci/nerd .
+clean:
+	rm --force --recursive .pytest_cache
+	rm --force --recursive $$(find . -name '__pycache__')
```

Por favor agrega la receta para borrar el cahé de las mutacioes:
```suggestion
	rm --force --recursive $$(find . -name '__pycache__')
	rm --force .mutmut-cache
```